### DISCUSSION TITLE
v0.5.1

### DISCUSSION THREAD

**Discussion by sheikhhaziq** [sheikhhaziq/gyawun_music#34] [Announcements]
## What's Changed
* Added Option to change app theme manually
* Added Chinese translation file by @thekocanlhk in https://github.com/sheikhhaziq/vibemusic/pull/19
* Added Arabic translation file by @thekocanlhk in https://github.com/sheikhhaziq/vibemusic/pull/20
* Added Korean translation file by @thekocanlhk in https://github.com/sheikhhaziq/vibemusic/pull/30
* Added Afrikaans translation file by @thekocanlhk in https://github.com/sheikhhaziq/vibemusic/pull/29
* Added Greek translation file by @thekocanlhk in https://github.com/sheikhhaziq/vibemusic/pull/28
* Update Japanese translations by @HiSubway in https://github.com/sheikhhaziq/vibemusic/pull/33

**Full Changelog**: https://github.com/sheikhhaziq/vibemusic/compare/0.4.2...v0.5.1

<hr /><em>This discussion was created from the release <a href='https://github.com/sheikhhaziq/vibemusic/releases/tag/v0.5.1'>v0.5.1</a>.</em>

**Comment by senolfndk** [comment]
can you add artist album

**Reply by sheikhhaziq** [reply]
Do you mean in search section

**Reply by senolfndk** [reply]
yes

**Reply by sheikhhaziq** [reply]
Sure, it'll be avialable in next release 🥳

**Reply by sheikhhaziq** [reply]
Album search is now available [here](https://github.com/sheikhhaziq/vibemusic/releases/tag/v0.6.0-beta.1) with beta release

**Reply by senolfndk** [reply]
thank you

**Comment by senolfndk** [comment]
Can you add artist and album tab?

**Reply by sheikhhaziq** [reply]
Those are already available in search tab

**Reply by senolfndk** [reply]
for example song artist album section like this one 
![Screenshot_20230110-154633_InnerTune](https://user-images.githubusercontent.com/52818836/211555916-8b7f68e1-510d-478c-9be8-20bc8ddc251a.jpg)

**Reply by sheikhhaziq** [reply]
It seem to be for favourite songs

**Reply by senolfndk** [reply]
save album

**Reply by sheikhhaziq** [reply]
I'll try to add it in next release.

**Comment by senolfndk** [comment]
Can you delete your search history?

**Reply by sheikhhaziq** [reply]
Yes, long press on history item to delete it

**Comment by senolfndk** [comment]
add widget to home screen

**Reply by sheikhhaziq** [reply]
I am currently working on that.

**Reply by senolfndk** [reply]
ok
